packages/pyright-internal/src/localization/simplified.nls.ca.json

```diff
@@ -0,0 +1,93 @@
+        "breakOutsideLoop": "\"interrupció (break)\" només es pot utilitzar dins d'un bucle \"mentre (while)\" o \"per a (for)\".",
+        "constructorNoArgs": "No s'esperava cap argument per al constructor \"{type}\".",
+        "continueOutsideLoop": "\"continue\" (continua) només es pot utilitzar dins d'un bucle \"mentre (while)\" o \"per a (for)\".",
+        "delTargetExpr": "L'expressió no es pot suprimir",
+        "duplicateParam": "Paràmetre duplicat \"{name}\"",
+        "expectedAssignRightHandExpr": "Expressió esperada a la dreta de \"=\"",
+        "expectedBinaryRightHandExpr": "Expressió esperada a la dreta de l'operador",
+        "expectedClassName": "Falta el nom de la classe",
+        "expectedCloseBrace": "Falta la clau de tancament \"}\"",
+        "expectedCloseBracket": "Falta el claudàtor de tancament \"]\"",
+        "expectedCloseParen": "Falta el parèntesi de tancament \")\"",
+        "expectedColon": "Falta dos punts \":\"",
+        "expectedDelExpr": "Falta l'expressió després de \"del\"",
+        "expectedElse": "Falta \"o sino\" (else)",
+        "expectedEqualityOperator": "Operador d'igualtat esperat, volies dir \"==\"?",
+        "expectedExpr": "Falta expressió",
+        "expectedFunctionName": "Falta el nom de la funció després de \"def\"",
+        "expectedIdentifier": "Falta l'identificador",
+        "expectedImport": "Falta \"import\"",
+        "expectedImportAlias": "Falta el símbol després de \"as\"",
+        "expectedImportSymbols": "S'esperava un o més noms de símbols després de la importació \"import\"",
+        "expectedIn": "Falta \"in\" (en)",
+        "expectedIndentedBlock": "Falta sagnat",
+        "expectedInExpr": "Falta l'expressió després d' \"in\" (en)",
+        "expectedMemberName": "Falta el nom del membre després de \".\"",
+        "expectedModuleName": "Falta el nom del mòdul",
+        "expectedNamedParameter": "El paràmetre de paraula clau ha d'anar després de \"*\"",
+        "expectedNewlineOrSemicolon": "Contingut addicional inesperat\nNomés pot haver-hi una instrucció per línia",
+        "expectedOpenParen": "Falta el parèntesi d'obertura \"(\"",
+        "expectedParamName": "Falta el nom del paràmetre",
+        "expectedSliceIndex": "Falta l'índex o la secció",
+        "expectedYieldExpr": "S'espera una expressió a la instrucció yield",
+        "globalReassignment": "\"{name}\" s'assigna abans de la declaració global",
+        "globalRedefinition": "\"{name}\" ja es va declarar global",
+        "importResolveFailure": "No s'ha pogut trobar el mòdul \"{importName}\".",
+        "importSymbolUnknown": "\"{name}\" no s'ha trobat al mòdul \"{moduleName}\"",
+        "inconsistentIndent": "El sagnat no coincideix amb la línia anterior",
+        "inconsistentTabs": "Ús incoherent de tabulacions i espais per a sagnats",
+        "instanceMethodSelfParam": "Els mètodes d'instància necessiten un paràmetre \"auto (self)\".",
```

Comment:
I think other languages didn't do this as the message relates to Python code that uses the English keywords. Might be worth a side-by-side review with say the Spanish for consistency here.